.github/workflows/pr-closed.yml

```diff
@@ -25,7 +24,8 @@ jobs:
               issue_number: context.issue.number,
               owner: context.repo.owner,
               repo: context.repo.repo,
-              body: `## Merge Successful
+              body: `
+              ## Merge Successful
```

Comment:
Are you sure that this won't add a new line before the headline?

Reply:
Sure